staging/src/k8s.io/apiserver/pkg/server/routes/helper.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routes
+
+import (
+	"net/http"
+)
+
+// RedirectTo redirects request to a certain destination.
+func RedirectTo(to string) func(http.ResponseWriter, *http.Request) {
```

Comment:
you can leave it in the old file, but make it lower case.

Reply:
I changed RedirectTo to lower case.